CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+# 1.0.3 / 04-02-2020
```

Comment:
I think we should bump the minor since there are a number of IMPROVEMENTS here and not just BUGFIXES